tasks/db_deploy/orca_sql.py

```diff
@@ -244,7 +244,7 @@ def schema_versions_data_sql() -> TextClause:
 
         -- Upsert the current version
         INSERT INTO schema_versions
-        VALUES (2, 'Updated recovery schema for v3.x of ORCA application', NOW(), True)
+        VALUES (3, 'Updated recovery schema for v3.x of ORCA application', NOW(), True)
```

Comment:
Change the comment to better reflect the change. Maybe `Updated recovery schema for ORCA v3.x to include multipart upload settings information.`

Reply:
Fixed